code/create/dta_dicom2spec_rules.py

```diff
@@ -1,158 +1,115 @@
-## rules for converting the dta-dicoms into a bids-dataset.
-
+# rules for converting the dta-dicoms into a bids-dataset.
 from datalad_hirni.support.BIDS_helper import apply_bids_label_restrictions
 
 
 def _guess_modality(record, dcm_dict):
 
-    protocol = record.get("ProtocolName", None)
+    # not sure, if the strings in the dcm-headers are always consistent,
+    # so we just ignore the capital-letters.
+    protocol = record.get("ProtocolName", None).lower()
     if protocol:
 
-        # BEGIN Additional rule for forrest-structural
-        # TODO: Probably to be moved to some rule enhancement
-        if "VEN_BOLD" in protocol:
-            # TODO: Not clear yet; swi might be considered a datatype rather than
-            # a modality by respective BIDS extension:
-            # https://docs.google.com/document/d/1kyw9mGgacNqeMbp4xZet3RnDhcMmf4_BmRgKaOkO2Sc
-            return "swi"
-
-        if "DTI_" in protocol:
-            # TODO: What actually is the relevant part of protocol here?
+        if "dti_"   in protocol:
             return "dwi"
-
-        if "60dir".lower() in protocol.lower():
+        if "60dir"  in protocol:
             return "dwi"
-        if "120dir".lower() in protocol.lower():
+        if "120dir" in protocol:
             return "dwi"
-
         if "field map" in protocol:
             return "fieldmap"
-        # END
 
         import re
-        prot_parts = re.split('_|-|\s', protocol.lower())
+        prot_parts = re.split('_|-|\s', protocol)
```

Comment:
This line is from the template. I am not good enough with regex, but should check at one point if there is a better fitting solution for DTA.